packages/cli/README.md

```diff
@@ -1,3 +1,10 @@
+# Overview
+
+[![oclif](https://img.shields.io/badge/cli-oclif-brightgreen.svg)](https://oclif.io)
+[![Version](https://img.shields.io/npm/v/smartthings-cli.svg)](https://npmjs.org/package/smartthings-cli)
+[![Downloads/week](https://img.shields.io/npm/dw/smartthings-cli.svg)](https://npmjs.org/package/smartthings-cli)
```

Comment:
The npm link here 404s

Reply:
I've updated these badges. I'm not really sure the ones we have make great sense but they do work now. It would be nice to add test coverage and a github releases link at some point at least.